warehouse/query-core/src/main/java/datawave/query/language/functions/jexl/Exclude.java

```diff
@@ -1,11 +1,12 @@
 package datawave.query.language.functions.jexl;
 
+import datawave.query.jexl.functions.QueryFunctions;
 import datawave.query.language.functions.QueryFunction;
 import datawave.query.search.WildcardFieldedFilter;
 
 public class Exclude extends AbstractEvaluationPhaseFunction {
     public Exclude() {
-        super("exclude");
+        super(QueryFunctions.EXCLUDE_FUNCTION);
```

Comment:
I would leave this the way it was.

Reply:
and ditto for the other language function classes

Reply:
I'll revert that to the way it was